package.json

```diff
@@ -51,6 +51,7 @@
     "build-examples": "rollup -c utils/build/rollup-examples.config.js",
     "dev": "concurrently --names \"ROLLUP,HTTP\" -c \"bgBlue.bold,bgGreen.bold\" \"rollup -c utils/build/rollup.config.js -w -m inline\" \"http-server -c-1 -p 8080\"",
     "dev-test": "concurrently --names \"ROLLUP,ROLLUPTEST,HTTP\" -c \"bgBlue.bold,bgRed.bold,bgGreen.bold\" \"rollup -c utils/build/rollup.config.js -w -m inline\" \"rollup -c test/rollup.unit.config.js -w -m inline\" \"http-server -p 8080\"",
+    "typecheck": "tsc -p utils/build/tsconfig.lint.json",
```

Comment:
I think this is a good change!

Do you think we need a separate `typecheck` script, though? This is covered in the `test-lint` script.

Reply:
whoops, didn't mean to commit this, this was a small helper function for myself to quickly run `tsc`. You are right it's covered by `test-lint`, will remove